### ISSUE TITLE
NTR: 3-hydroxyisobutyrate dehydrogenase deficiency (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8191]
New term request 3-hydroxyisobutyrate dehydrogenase deficiency (segal_070716193119):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000214
- label: Decreased 3-hydroxyisobutyrate dehydrogenase activity
- parents: [Abnormal circulating enzyme concentration or activity (HP:0012379)](https://hpo.jax.org/app/browse/term/HP:0012379)
- pmid: 
- def: The activity of 3-hydroxyisobutyrate dehydrogenase in the blood circulation is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8191
- synonyms:

**Comment by MickeySegal** [comment]
Shouldn't the label be "Reduced circulating 3-hydroxyisobutyrate dehydrogenase activity", the style used in https://hpo.jax.org/browse/term/HP:6000215?
